flagging_site/blueprints/api.py

```diff
@@ -12,6 +12,7 @@
 from ..data.model import reach_3_model
 from ..data.model import reach_4_model
 from ..data.model import reach_5_model
+from flagging_site.config import API_MAX_HOURS
```

Comment:
call from `current_app.config['API_MAX_HOURS']` instead.

Reply:
Yes, I see. Fixed both of these changes!